### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: book/02-using-atom/sections/01-packages-themes.asc [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 
-In fact, there are more than 70 packages that comprise all of the functionality that is available in Atom by default. As some examples, the https://github.com/atom/welcome[Welcome dialog] that you see when you first start it, the https://github.com/atom/spell-check[spell checker], the https://github.com/atom/atom-dark-ui[themes] and the https://github.com/atom/fuzzy-finder[fuzzy finder] are all packages that are seperately maintained and all use the same APIs that you have access to, as we'll see in great detail in <<_hacking_atom>>.
+In fact, there are more than 70 packages that comprise all of the functionality that is available in Atom by default. As some examples, the https://github.com/atom/welcome[Welcome dialog] that you see when you first start it, the https://github.com/atom/spell-check[spell checker], the https://github.com/atom/atom-dark-ui[themes] and the https://github.com/atom/fuzzy-finder[fuzzy finder] are all packages that are separately maintained and all use the same APIs that you have access to, as we'll see in great detail in <<_hacking_atom>>.
```
Comment (pborreli) [inline book/02-using-atom/sections/01-packages-themes.asc]: seperately -> separately (as you can't see it with the diff tool)

**Comment by schacon** [comment]
Wow, that was super fast. Thanks!

Do you want to me keep this open for a bit in case you find more, or do you want me to merge this now?

**Comment by pborreli** [comment]
I'm done reading everything, I will come back time to time to check updates as always don't worry

**Comment by benogle** [comment]
> I'm done reading everything

:book: :bug:

File: book/02-using-atom/sections/02-text.asc [hunk 2]
```diff
@@ -150,7 +150,7 @@ The whitespace helper tools are maintained as a separate package and so the sett
 .Managing your whitespace settings
 image::images/whitespace.png[whitespace settings]
 
-Note that the ``Remove Training Whitespace'' option is on by default. This means that every time you save any file opened in Atom, it will strip all trailing whitesapce from the file. If you want to disable this, go to the `whitespace` package in your settings panel and uncheck that option.
+Note that the ``Remove Training Whitespace'' option is on by default. This means that every time you save any file opened in Atom, it will strip all trailing whitespace from the file. If you want to disable this, go to the `whitespace` package in your settings panel and uncheck that option.
```
Comment (postcasio) [inline book/02-using-atom/sections/02-text.asc]: Training -> Trailing

**Comment by postcasio** [comment]
Oops, I was too late

File: book/02-using-atom/sections/02-text.asc (hunk 2 above)
Comment (pborreli) [inline book/02-using-atom/sections/02-text.asc]: good catch, you should open a PR and proof read the docs now :smile:

**Pull request merged**